time.lua

```diff
@@ -0,0 +1,35 @@
+--- Functions and properties for dealing with ticks.
+--- @class flib_time
+local flib_time = {}
+
+flib_time.second = 60
```

Comment:
`second`, etc. are not very descriptive. A second of what? I would change these to `second_ticks` so it's perfectly clear what they are.

Reply:
Actually the more common expression is ticks_per_second. And more descriptive also.